frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx

```diff
@@ -55,7 +55,12 @@ function cardContent({ robot }: RobotProps) {
                     <RobotStatusChip status={robot.status} />
                 </VerticalContent>
                 <VerticalContent $alignItems="end">
-                    <PressureStatusView pressure={robot.pressureLevel} />
+                    <PressureStatusView
+                        pressure={robot.pressureLevel}
+                        pressureStatus={PressureStatus.Normal}
+                        upperPressureWarningThreshold={robot.model.upperPressureWarningThreshold ?? 80}
```

Comment:
Should this be where the default value lives? should be defined somewhere and not in the function

Reply:
This value will only be utilized incase the value is somehow null. It is the same value in Initdb.cs

Reply:
It should still be defined as a constant, e.g defaultUpperPressureWarningThreshold and used instead of some "loose" value in code

Reply:
Do we want default values? Isn't it better to just show the pressure value instead of including potentially misleading warnings?